website/source/docs/configuration/storage/google-cloud-storage.html.md

```diff
@@ -7,7 +7,7 @@ description: |-
   Google Cloud Storage.
 ---
 
-# Google Cloud Storage Backend
+# Google Cloud Storage Storage Backend
```

Comment:
That heading tho

Reply:
Yeaaaa. But it's called "Google Cloud Storage", and it's a Vault "Storage Backend", thus the double storage party